src/Wpf.Ui/Controls/NavigationView/INavigationView.cs

```diff
@@ -36,22 +36,22 @@ public interface INavigationView
     /// <summary>
     /// Gets the collection of menu items displayed in the NavigationView.
     /// </summary>
-    IList MenuItems { get; }
+    ObservableCollection<object> MenuItems { get; }
 
     /// <summary>
     /// Gets or sets an object source used to generate the content of the NavigationView menu.
     /// </summary>
-    object? MenuItemsSource { get; set; }
+    ObservableCollection<object> MenuItemsSource { get; set; }
 
     /// <summary>
     /// Gets the list of objects to be used as navigation items in the footer menu.
     /// </summary>
-    IList FooterMenuItems { get; }
+    ObservableCollection<object> FooterMenuItems { get; }
 
     /// <summary>
     /// Gets or sets the object that represents the navigation items to be used in the footer menu.
     /// </summary>
-    object? FooterMenuItemsSource { get; set; }
+    ObservableCollection<object> FooterMenuItemsSource { get; set; }
```

Comment:
ItemSource of other WPF controlls does not directly bound to ObservableCollection
https://learn.microsoft.com/pl-pl/dotnet/api/system.windows.controls.itemscontrol.itemssource?view=windowsdesktop-8.0#system-windows-controls-itemscontrol-itemssource

Reply:
![image](https://github.com/lepoco/wpfui/assets/7411687/5078d6ee-c3ee-4402-8368-331dc3e09839)
I seems they are suggesting to use ObservableCollection. According to the documentation.

Reply:
This is a recommendation to the end user, which does not mean that the library should enforce this type of collection